### ISSUE TITLE
Set default response type to text/turtle for the root folder

### ISSUE THREAD

**Issue by Otto-AA** [solid/node-solid-server#1171]
## Please describe what you did in reproducible steps
Fetch the root folder of a pod on a v5.0.0 server without specifying the "Accept" header.
(e.g. GET https://testomat.solid.open.ac.uk/, `Accept: */*`)

## How did it work with 4.x series servers?
It returned a text/turtle file.

## What happened when you tried the same with the 5.x series server?
It returns a text/html file.

## Any material that will help, logs, error messages, etc.
This is a follow up question of [this issue](https://github.com/solid/node-solid-server/issues/1120).

As far as I understand the [spec](https://github.com/solid/solid-spec/blob/master/api-rest.md#reading-resources) it should return a `text/turtle` response by default if a container (folder) is requested. This also was the case for the root folder in 4.x, but changed with 5.0.0.

> IMPORTANT: a default Content-Type: text/turtle will be used for requests for RDF resources or views (containers) that do not have an Accept header.

[Here](https://github.com/jeff-zucker/solid-file-client/issues/33#issuecomment-481823805) is a more precise explanation of the behavior of the different versions, if the above was not expressive enough.

**Comment by Otto-AA** [comment]
I've noticed another difference from 5.x to older versions regarding the root container:
POST to the root responds with 403 (e.g. creating a new folder or file in the root).
GET of an item in the root responds with 403 if it doesn't exist (e.g. GET {root}/non-existent/).

Both work with the same settings in my private folder. In the profile I've set read/write/append/control access. I sadly don't have the ability to look into server settings, so maybe that could be the reason too.

Is this related to this issue, or should I open a new one?

![image](https://user-images.githubusercontent.com/21040751/55935481-05fc8200-5c34-11e9-9a04-424c8e577864.png)
